src/java.base/share/classes/java/io/File.java

```diff
@@ -1936,6 +1944,10 @@ public static File createTempFile(String prefix, String suffix)
      * systems, alphabetic case is significant in comparing pathnames; on
      * Microsoft Windows systems it is not.
      *
+     * @apiNote This method only compares the abstract pathnames;
+     *          it does not access the file system and the file is not required
+     *          to exist.
```

Comment:
Rather than API note  hen it might be better to take the second part of the proposed note and add it to the first paragraph of the method description. Can you try that and see how it reads?